src/lib/activity-store.js

```diff
@@ -4,10 +4,14 @@ import Elasticsearch from '@elastic/elasticsearch';
 class ActivityStore {
 
   constructor(esIndex){
-    this.user = "test";
-    this.pass = "test";
+    this.user = process.env.ELASTICSEARCH_USERNAME || "";
+    this.pass = process.env.ELASTICSEARCH_PASSWORD || "";
     this.esIndex = esIndex;
-    this.client = new Elasticsearch.Client({ node: 'http://localhost:9200' });
+    let client_options = { node: process.env.ELASTICSEARCH_URL || 'http://localhost:9200' };
+    if (this.user || this.pass) {
```

Comment:
&& , rather than || I would have thought you'd always need both present for auth

Reply:
dunno if you are allowed blank passwords but sure, let's say it should be